Dockerfile.amd64

```diff
@@ -46,6 +40,16 @@ RUN apt-get update && apt-get install -y \
     procps \    
     ca-certificates
 
+RUN echo 'APT::Default-Release "stable";' > /etc/apt/apt.conf.d/99defaultrelease
```

Comment:
Installing a backport is a bit unusual so please add a comment explaining it (as we had for the 1.6.2 backport)